addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -70,17 +70,38 @@ weSnippetEditor.Class.include({
                 size: 'medium',
                 title: _t("Google Map API Key"),
                 buttons: [
-                    {text: _t("Save"), classes: 'btn-primary', close: true, click: async () => {
-                        const newAPIKey = dialog.$('#api_key_input').val() || false;
-                        await this._rpc({
-                            model: 'website',
-                            method: 'write',
-                            args: [
-                                [websiteId],
-                                {google_maps_api_key: newAPIKey},
-                            ],
-                        });
-                        invalidated = true;
+                    {text: _t("Save"), classes: 'btn-primary', close: false, click: async (event) => {
```

Comment:
Name the `event` variable `ev` instead. We avoid to use `event` as it is the name of a global variable on Chrome so this has been the source of mistakes more than once :wink: